unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/BuildModeHUDController.cs

```diff
@@ -7,10 +7,26 @@
 
 public class BuildModeHUDController : IHUD
 {
-    public event Action OnChangeModeAction, OnTranslateSelectedAction, OnRotateSelectedAction, OnScaleSelectedAction, OnResetAction, OnDuplicateSelectedAction, OnDeleteSelectedAction;
-    public event Action OnEntityListVisible, OnStopInput,OnResumeInput, OnTutorialAction,OnPublishAction;
+    public event Action OnChangeModeAction,
+                        OnTranslateSelectedAction,
+                        OnRotateSelectedAction,
+                        OnScaleSelectedAction,
+                        OnResetAction,
+                        OnDuplicateSelectedAction,
+                        OnDeleteSelectedAction;
```

Comment:
```suggestion
    public event Action OnChangeModeAction;
    public event OnTranslateSelectedAction;
    public event OnRotateSelectedAction;
    public event OnScaleSelectedAction;
    public event OnResetAction;
    public event OnDuplicateSelectedAction;
    public event OnDeleteSelectedAction;
```

We dont use this kind of declaration style